Code/GraphMol/FMCS/FMCS.h

```diff
@@ -116,7 +116,7 @@ struct MCSParameters {
         AtomTyper(MCSAtomCompareElements),
         BondTyper(MCSBondCompareOrder),
         CompareFunctionsUserData(0),
-        ProgressCallback(MCSProgressCallbackTimeout),
```

Comment:
This is a good solution.  If you want to see the details of what is going on, take a deep breath and read this:

```
https://www.akkadia.org/drepper/dsohowto.pdf
```

If you like beer, have a lot of it ready :)